### PR TITLE
Go client

### PR DESCRIPTION
**Pull request by hao-liu-1117**
Hi!

I've add command line client in go version. Could you help me review on that?

Thanks!

### CODE REVIEW

File: src/go-client/client.go [hunk 1]
```diff
@@ -0,0 +1,260 @@
...
+		fmt.Println("Log in before following someone.")
+		return
+	}
+	if len(to_follow) == 0 {
+		fmt.Println("Can not follow an empty username.")
+		return
+	}
+	// Construct caw::FollowRequest
+	req := &caw.FollowRequest{Username: username, ToFollow: to_follow}
+	rep := new(caw.RegisteruserRequest)
```
Comment (CallMeMajorTom) [inline src/go-client/client.go:130]: I think reply should be `caw.FollowReply` type?

File: src/go-client/client.go [hunk 2]
```diff
@@ -0,0 +1,260 @@
...
+
+	flag.Parse()
+
+	conn, err := grpc.Dial("localhost:50000", grpc.WithInsecure(), grpc.WithBlock())
+	if err != nil {
+		fmt.Println("failed to connect")
+		return
+	}
+	defer conn.Close()
+	c := faz.NewFazServiceClient(conn)
```
Comment (CallMeMajorTom) [inline src/go-client/client.go]: Probably use a more readable name for fazclient?

**Review by CallMeMajorTom — APPROVED** [review]
LGTM, left some comments. I also think we shouldn't include the binary file?

File: src/go-client/client.go (hunk 2 above)
Comment (hao-liu-1117) [reply src/go-client/client.go]: I agree, I'll rename it.

File: src/go-client/client.go (hunk 1 above)
Comment (hao-liu-1117) [reply src/go-client/client.go:130]: I agree, but caw.FollowReply contains no data. So I just use a message to verify if server works in logic (it seems ugly though...).

**Comment by hao-liu-1117** [comment]
I'll delete binary file in src/ .

**Pull request merged**